README.md

```diff
@@ -0,0 +1,156 @@
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hello Microverse]** is a simple project based on HTML CSS Linters and Gitflow. And also it the first week project of microverse
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Hello Microverse Project is a project which is  based on HTML CSS GithubFlow and Linters
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://en.wikipedia.org/wiki/HTML">HTML</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The key features for this projects were HTML CSS Linters.Describe between 1-3 key features of the application.
+
+- **[HTML]**
+- **[CSS]**
+- **[Linters]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Node.js to install from the Node.js official Website.
+
+[Node.js](https://nodejs.org/en)
+
+Chrome web browser
+
+[Chrome](https://www.google.com/chrome/?brand=JJTC&gclid=CjwKCAjw9J2iBhBPEiwAErwpeTs4U9U2QjXaHjW4P3xWKxrsnLr6U3_X1J0MOUPoGHV4U46qNcUhLxoC7XQQAvD_BwE&gclsrc=aw.ds)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/zuhebahmed88091/Hello_Microverse_Project.git
+  cd my-project
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  npm install
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+Example command:
+
+```sh
+  npx hint .
+  npx stylelint "**/*.{css/scss}"
+```
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: [@zuhebahmed88091](https://github.com/zuhebahmed88091)
+- Twitter: [@ZuhebAhmed88091](https://twitter.com/ZuhebAhmed88091)
+- LinkedIn: [Zuheb Ahmed](https://www.linkedin.com/in/zuheb-ahmed/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please share your thoughts and give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse to give an opportunity to do the project 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The link to your MIT license is broken because the link specified is wrong. I think the right link should be [MIT](./MIT.md). Kindly check it out 👍🏽